packages/vite-plugin-checker/src/logger.ts

```diff
@@ -1,15 +1,13 @@
 import chalk from 'chalk'
-import fs from 'fs'
-import { createRequire } from 'module'
-import os from 'os'
+import os from 'node:os'
 import strip from 'strip-ansi'
 import * as _vscodeUri from 'vscode-uri'
 
 // hack to compatible with Jiti
 // see details: https://github.com/fi3ework/vite-plugin-checker/issues/197
 // @ts-expect-error
 const URI = _vscodeUri?.default?.URI ?? _vscodeUri.URI
```

Comment:
this "hack" should be moved to places where we import `vscode-uri`, as its no longer used in here we should remove it from this file